pkg/validation/exception/validate.go

```diff
@@ -25,6 +30,37 @@ func Validate(ctx context.Context, logger logr.Logger, polex *kyvernov2alpha1.Po
 	} else if opts.Namespace != "" && opts.Namespace != polex.Namespace {
 		warnings = append(warnings, namespacesDontMatch)
 	}
+	var errors []error
+	err := validateVariables(polex)
```

Comment:
Why not have it inside `polex.Validate()`?

Reply:
Yeah, i should do that